src/System.Net.Http/tests/FunctionalTests/DefaultCredentialsTest.cs

```diff
@@ -14,7 +14,7 @@ namespace System.Net.Http.Functional.Tests
     using Configuration = System.Net.Test.Common.Configuration;
 
     // TODO: #2383 - Consolidate the use of the environment variable settings to Common/tests.
-    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "dotnet/corefx #20010")]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "dotnet/corefx #20010")]
```

Comment:
[ActiveIssue] is not supported for class declarations, but it should be. I have just opened this issue: https://github.com/dotnet/buildtools/issues/1508 to track this. I should be solving that soon. When that is solved then I can go over and do that cleanup on another PR.